src/main.rs

```diff
@@ -177,6 +177,10 @@ struct Args {
     )]
     policy_timeout_ms: Option<Duration>,
 
+    /// If provided, enables interfacing with a secrets backend for secret retrieval
```

Comment:
It just matches our current precedent for the policy service topic which was the main reason I went in this direction, I think having a default topic prefix would be fine the main case I want to avoid is passing a topic prefix and forgetting to enable the support

Reply:
Ah OK that's pretty reasonable -- having  the one can certainly (and does) work with policy service